docs/src/main/asciidoc/doc-create-tutorial.adoc

```diff
@@ -37,18 +37,18 @@ Copy `docs/src/main/diataxis/_templates/template-tutorial.adoc` from the Quarkus
 
 == Update the document header
 
-[source,asciidoc]
+[source,AsciiDoc]
```

Comment:
I don't think we want that change (and all the other identical changes below).

Reply:
From what I can see, a search and replace `[source,AsciiDoc` (without the `]`) -> `[source,asciidoc` should fix it.

Reply:
oh! you're right. crap. Good catch.